Mongoose Syntax Highlighting and Snippets For Sublime

PranavSathy opened the pull request:
Better CoffeeScript with Mongoose Snippets & Syntax Highlighting

FichteFoll commented:
Using branches for releases is deprecated and won't be accepted for new packages anymore: https://packagecontrol.io/docs/submitting_a_package#Step_4

PranavSathy commented:
Thanks for the prompt response! I removed the package.json, updated to tag based releases and have created the appropriate tag (v0.1.0) for my release.
